src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -605,87 +607,83 @@ function MoneyRequestConfirmationList(props) {
             shouldShowTextInput={false}
             shouldUseStyleForChildren={false}
             optionHoveredStyle={canModifyParticipants ? styles.hoveredComponentBG : {}}
-            footerContent={(!isEmpty(props.iou.id) || props.isEditingSplitBill) && footerContent}
-            listStyles={props.listStyles}
+            footerContent={(!!iou?.id || isEditingSplitBill) && footerContent}
+            listStyles={listStyles}
             shouldAllowScrollingChildren
         >
-            {props.isDistanceRequest && (
+            {isDistanceRequest && (
                 <View style={styles.confirmationListMapItem}>
-                    <ConfirmedRoute transaction={props.transaction} />
+                    <ConfirmedRoute transaction={transaction ?? ({} as OnyxTypes.Transaction)} />
```

Comment:
Is it possible instead of assertion:
1) Update `ConfirmedRoute` typing?
2) Update condition to `{isDistanceRequest && !!transaction &&`